### PR TITLE
[no ticket]updating hail version

### PR DESCRIPTION
**Pull request by xuscrbw**
\<your comments for this PR go here\>
Generating custom image
---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by xuscrbw** [comment]
multi-test

**Comment by xuscrbw** [comment]
jenkins retest

File: jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ set -e -x
 terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.0.4"
 terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.0.5"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.0.5"
-terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.8"
+terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.9"
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.0.8"
 terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.0.12"
```
Comment (calbach) [inline jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh]: Any chance the AoU 2.0.13 image can hop on this train (Dataproc and GCE)? Not urgent, so if the Hail fix is time sensitive, please ignore.

https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-aou/CHANGELOG.md#2013---2022-03-18

**Comment by xuscrbw** [comment]
multi-test

File: jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh (hunk 1 above)
Comment (xuscrbw) [reply jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh]: Yep, we can bump this as well. I'm noticing that we store an old image on the leo image as well, should we also bump this version up to 2.0.12?

Comment (calbach) [reply jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh]: 2.0.11 is in active use, so please keep that as the "old" version. 2.0.12 did not wind up being used by us, so no need to cache it. Thank you!

**Comment by xuscrbw** [comment]
PR covered by this: https://github.com/DataBiosphere/leonardo/pull/2590/files

**Comment by xuscrbw** [comment]
PR covered by this: https://github.com/DataBiosphere/leonardo/pull/2590/files

**Comment by Qi77Qi** [comment]
replaced by https://github.com/DataBiosphere/leonardo/pull/2590

**Pull request closed without merging**